src/gui/transferlistmodel.cpp

```diff
@@ -406,7 +406,7 @@ QColor getColorByState(BitTorrent::TorrentState state)
         return QColor(255, 0, 0); // red
     default:
         Q_ASSERT(false);
-        return QColor(255, 0, 0); // red
+        return {255, 0, 0}; // red
```

Comment:
you might also apply to this whole function, I mean to other `return QColor(...);`

Reply:
I made a search in the project and found a whole lot more of these but i don't know if there's a reason clang-tidy didn't mention them... added new commit with them.